dds/src/implementation/rtps/types.rs

```diff
@@ -395,9 +386,9 @@ impl From<Count> for i32 {
 /// The following values are reserved by the protocol: LOCATOR_INVALID LOCATOR_KIND_INVALID LOCATOR_KIND_RESERVED LOCATOR_KIND_UDPv4 LOCATOR_KIND_UDPv6 LOCATOR_ADDRESS_INVALID LOCATOR_PORT_INVALID
 #[derive(Clone, Copy, PartialEq, Eq, Debug, serde::Serialize, serde::Deserialize)]
 pub struct Locator {
-    pub kind: LocatorKind,
-    pub port: LocatorPort,
-    pub address: LocatorAddress,
+    kind: LocatorKind,
+    port: LocatorPort,
+    address: LocatorAddress,
 }
 type LocatorKind = i32;

```

Comment:
These types here would deserve to become more than typedefs as well

Reply:
Yes. As well as Sequencenumber, Fragmentnumber, GroupDigest and others